curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md

```diff
@@ -8,36 +8,32 @@ forumTopicId: 301140
 
 ## Description
 <section id='description'>
-Making images responsive with CSS is actually very simple. Instead of applying an absolute width to an element:
-<code>img { width: 720px; }</code>
-You can use:
+Making images responsive with CSS is actually very simple. You just need to add these properties to an image:
 
 ```css
 img {
   max-width: 100%;
-  display: block;
   height: auto;
 }
 ```
 
-The <code>max-width</code> property of 100% scales the image to fit the width of its container, but the image won't stretch wider than its original width. Setting the <code>display</code> property to block changes the image from an inline element (its default), to a block element on its own line. The <code>height</code> property of auto keeps the original aspect ratio of the image.
+The `max-width` of `100%` will make sure the image is never wider than the container it is in, and the `height` of `auto` will make the image keep its original aspect ratio.
```

Comment:
How about this?
```suggestion
`max-width:100%;` will make sure the image is never wider than the container it is in, and `height: auto;` will make the image keep its original aspect ratio.
```

Reply:
Either way would work here, but I prefer how it is now - seems more readable.